app/models.py

```diff
@@ -340,3 +340,50 @@ class UserSkill(db.Model): #pylint: disable=no-init,too-few-public-methods
     user_id = Column(types.Integer, ForeignKey('users.id'), nullable=False)
 
     constraint = UniqueConstraint('user_id', 'name')
+
+class Event(db.Model):
+    '''
+    An event that shows up in the activity feed for a deployment.
+    '''
+
+    __tablename__ = 'events'
+    id = Column(types.Integer, autoincrement=True, primary_key=True)  #pylint: disable=invalid-name
+    created_at = Column(types.DateTime(), default=datetime.datetime.now)
+    updated_at = Column(types.DateTime(), default=datetime.datetime.now,
+                        onupdate=datetime.datetime.now)
+    type = Column(types.String)
+
+    user_id = Column(types.Integer, ForeignKey('users.id'))
```

Comment:
Yeah, it's not the worst thing if it is.  We could do the `UserEvent` subclass you're talking about, and have `deployment` live on the top-level class.  We'd pre-populate it with the user's `deployment` on creation.